agent/config/types.go

```diff
@@ -332,4 +333,7 @@ type Config struct {
 
 	// External specifies whether agent is running on external compute capacity (i.e. outside of aws).
 	External BooleanDefaultFalse
+
+	// PrimaryIPv4VPCCIDR stores the primary IPv4 CIDR of the VPC in which agent is running
+	PrimaryIPv4VPCCIDR *net.IPNet
```

Comment:
can you add a comment here saying this field is currently only populated for windows and used for windows task networking setup? just to avoid confusion in the future